src/main/java/teammates/common/util/Const.java

```diff
@@ -96,6 +96,8 @@
     public static final Instant TIME_REPRESENTS_NOW;
     public static final Instant TIME_REPRESENTS_DEFAULT_TIMESTAMP;
 
+    public static final String TEAMMATE_HOME_PAGE_LINK = "<a href=\"teammatesv4.appspot.com\">TEAMMATES</a>";
```

Comment:
Now you're hardcoding it here. The value should be read from `Config.java` instead.

Reply:
Also, note that this would link to `https://mail.google.com/mail/u/0/teammatesv4.appspot.com` (or similar depending on email client). Always remember to include the protocol in the `href` for absolute links.

Reply:
ok i will change it

Reply:
The point is that it should be configurable via `build.properties` since it varies from deployment to deployment. E.g. when deploying to a staging server, the URL should point to that staging server, not the live server.